src/_styles/_variables.scss

```diff
@@ -63,3 +63,27 @@ $size-unit: 4px;
 
 $grid-columns: 12;
 $font-size: 14px;
+
+// Font-size
+
+$size-paragraph: 14px;
+$size-label: 14px;
+$size-label-sm: 11px;
+
+@media (max-width: $breakpoint-sm) {
+  $size-h1: 60px;
```

Comment:
Property `line-height` also changes with font size. Let's add dedicated `line-height` to each size variables. You can use mixins.

```
@mixin header-large-font {
  font-size: 56px;
  line-height: 64px;

  font-family: $font-family-regular;
}
```

Reply:
Done [0636669](https://github.com/AnastasiSo/Portfolio-Project/pull/14/commits/06366690a473de707cbdd14dfb9e72342e9ce536)✔.